src/Parser/DeathKnight/Unholy/Modules/Items/ColdHeartEfficiency.js

```diff
@@ -21,49 +21,93 @@ class ColdHeartEfficiency extends Analyzer {
   totalColdHeartCasts = 0;
   correctColdHeartCasts = 0;
   buffColdHeart = 0;
+  
   unholyStrengthStart = 0;
   unholyStrengthRemaining = 0;
+  
+  concordanceStart = 0;
+  concordanceRemaining = 0;
+  
+  khazgorothStart = 0;
+  khazgorothRemaining = 0;
+  
+  timeAtMaxStacksStart = 0;
+  timeAtMaxStacks = 0;
```

Comment:
Several of the above variables are only assigned and then immediately used in the `on_byPlayer_cast` block. As such, they should be defined in that block (as consts) instead of here. 

I'm talking about `timeAtMaxStacks` `khazgorothRemaining` `concordanceRemaining` `unholyStrengthRemaining`